src/combineReducers.js

```diff
@@ -115,13 +115,13 @@ export default function combineReducers(reducers) {
       finalReducers[key] = reducers[key]
     }
   }
-  var finalReducerKeys = Object.keys(finalReducers)
+  const finalReducerKeys = Object.keys(finalReducers)
 
   if (NODE_ENV !== 'production') {
     var unexpectedKeyCache = {}
```

Comment:
A fix I had was declaring `let unexpectedKeyCache` above the conditional check. But I don't know if you guys want an undefined variable floating around in production.